modules/SitecoreImageBuilder/SitecoreImageBuilder.psm1

```diff
@@ -703,38 +703,59 @@ function Get-LatestSupportedVersion
     # load Windows image specifications
     $specs = Get-BuildSpecifications -Path (Join-Path $PSScriptRoot "\..\..\windows")
 
-    # find all Sitecore versions
-    $versions = $specs | Where-Object { $_.Tag -like "sitecore-*:*windowsservercore-*" } | Select-Object -ExpandProperty Tag
-    $versions = $versions | ForEach-Object {
-        $_.Substring($_.IndexOf(':') + 1)
-    }
-
-    $versions = $versions | ForEach-Object {
-        $_.Substring(0, $_.IndexOf('-'))
-    }
-
-    $versions = $versions | Sort-Object -Unique -Descending
-
-    # pick latest Sitecore version
-    $sitecore = $versions | Select-Object -First 1
+    # get the latest version number for Sitecore
+    $sitecore = Get-LatestVersionNumberForTag -specs $specs -tag "sitecore-*:*windowsservercore-*"
 
     # pick latest 'windowsservercore' LTSC version
     $windowsServerCore = (Get-SupportedWindowsVersions | Where-Object { $_ -like "ltsc*" } | Select-Object -First 1)
 
     # pick latest 'nanoserver' version matching latest 'windowsservercore' LTSC version
     $nanoserver = (Get-WindowsServerCoreToNanoServerVersionMap)[$windowsServerCore]
 
+    # get the latest version for redis
+    $redis = Get-LatestVersionNumberForTag -specs $specs -tag "sitecore-redis*:*windowsservercore-*"
+
     Write-Output (New-Object PSObject -Property @{
             Sitecore          = $sitecore;
             WindowsServerCore = $windowsServerCore;
             NanoServer        = $nanoserver;
+            Redis            = $redis;
         })
 }
 
+function Get-LatestVersionNumberForTag
+{
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $true)]
+        [array]$specs
+        ,
+        [Parameter(Mandatory = $true)]
+        [string]$tag
```

Comment:
Please correct the casing of the parameter, including it's usage

Reply:
Done!